src/terraform/install.sh

```diff
@@ -44,26 +44,9 @@ if [ "$(id -u)" -ne 0 ]; then
     exit 1
 fi
 
-# Get central common setting
-get_common_setting() {
```

Comment:
@Chuxel Any concerns with removing this? 
We already have the correct variables set in this `install.sh` file.

Reply:
If we happen to need this `settings.env` file, we should think of migrating it from the old repo.